backend/market/models.py

```diff
@@ -8,51 +8,81 @@
 
 class Offer(models.Model):
     class Meta:
-        constraints = [models.UniqueConstraint(fields=["user", "sublet"], name="unique_offer")]
+        constraints = [models.UniqueConstraint(fields=["user", "item"], name="unique_offer_market")]
 
-    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="offers_made")
-    sublet = models.ForeignKey("Sublet", on_delete=models.CASCADE, related_name="offers")
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="offers_made_market")
+    item = models.ForeignKey("Item", on_delete=models.CASCADE, related_name="offers")
     email = models.EmailField(max_length=255, null=True, blank=True)
     phone_number = PhoneNumberField(null=True, blank=True)
     message = models.CharField(max_length=255, blank=True)
     created_date = models.DateTimeField(auto_now_add=True)
 
     def __str__(self):
-        return f"Offer for {self.sublet} made by {self.user}"
+        return f"Offer for {self.item} made by {self.user}"
 
 
-class Amenity(models.Model):
+class Category(models.Model):
+    '''
```

Comment:
nit: remove this comment. feel free to post it in our slack chat or smth if u want to keep it around somewhere but lets not have dead code

Reply:
I kept it in so we remember which Categories to create when deploying the app. But yeah keeping it in Slack will be cleaner.